x-pack/plugins/security_solution/public/common/components/event_details/table/action_cell.tsx

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback, useState, useRef } from 'react';
+import { getDraggableId } from '@kbn/securitysolution-t-grid';
+import { HoverActions } from '../../hover_actions';
+import { useActionCellDataProvider } from './use_action_cell_data_provider';
+import { EventFieldsData } from '../types';
+import { useGetTimelineId } from '../../drag_and_drop/draggable_wrapper_hover_content';
+import { ColumnHeaderOptions } from '../../../../../common/types/timeline';
+import { BrowserField } from '../../../containers/source';
+
+interface Props {
+  contextId: string;
+  data: EventFieldsData;
+  disabled?: boolean;
+  eventId: string;
+  fieldFromBrowserField: Readonly<Record<string, Partial<BrowserField>>>;
```

Comment:
I make this to be optional in my PR  as I reused this in overview tab, and some of the custom fields  doesn't have this.

Reply:
Yea, that makes sense! 😄